corehq/apps/app_manager/models.py

```diff
@@ -5761,7 +5815,7 @@ def get_profile_setting(self, s_type, s_id):
     def has_careplan_module(self):
         return any((module for module in self.modules if isinstance(module, CareplanModule)))
 
-    @quickcache(['self.version'])
+    @quickcache(['self._id', 'self.version'])
```

Comment:
interesting, so this was using a single cache key for all apps HQ with the same version... that's baaaad